compiler/cpp/src/thrift/generate/t_cl_generator.cc

```diff
@@ -221,11 +221,11 @@ void t_cl_generator::package_def(std::ostream &out) {
     }
     out << ")";
   }
-  out << ")" << endl << endl;
+  out << ")\n\n";
```

Comment:
in cases like this, the usage of `endl`/`std::endl` makes it much more obvious how many lines we are adding while `\n` being part of the longer string just meshes everything together.

I'd suggest still explicitly adding every `\n` individually, like this:
```suggestion
  out << ")" << "\n" << "\n";
```

but I don't feel strongly either way

Reply:
@fishy Thanks for the review and feedback.

Although I do understand the suggestion I specifically went through the effort to combine these. It does sort of stand out, but some editors color the `\n` different and I deem that good enough. I find myself looking at the generated code much more often than the generator code thus there I quickly see how much (or few) newlines are added. 

If there is enough pushback against the combination I can revert that combination change and (force)push the updates. 

I don't think speed matters that much but I guess calling the `operator<<` multiple times, more than the needed amount of time, would have a performance and speed implication.

Reply:
I'm slightly with @fishy here, as my comment in https://github.com/apache/thrift/pull/2943#issuecomment-1986363604 was actually going in the same direction: Having dedicated calls to `<< "\n"` jumps much more to the eye and conveyes how many newlines are added.

If you would be willing to change that, to make every newline jump out more, I'm two thumbs up for this PR! But without that change, I'm still ok to merge, albeit with minor hesitation...

Reply:
@emmenlau Thank you again for the feedback.
I think there is enough reason to put back the `<< "\n` thus I will put in the effort (tonight) to 'revert' that change.